experimental/DrinfeldHeckeAlgebras/src/AlternatingBilinearForm.jl

```diff
@@ -0,0 +1,74 @@
+#######################################
+# Methods for creating and applying alternating bilinear forms
+#
+# Cassandra Koenen, 2025
+#######################################
+
+#######################################
+# Struct and constructors
+#######################################
+
+struct AlternatingBilinearForm{T <: RingElem}
+  matrix::MatElem{T}
+
+  function AlternatingBilinearForm(m::MatElem{T}) where T <: RingElem
+    if ncols(m) != nrows(m)
+      throw(ArgumentError("Matrix representing an alternating bilinear form must be quadratic")) 
+    end
+  
+    if transpose(m) != -m || !is_zero(diagonal(m))
+      throw(ArgumentError("Input matrix must be skew symmetric with zero diagonal")) 
+    end
+  
+    return new{T}(m)
+  end
+end
+
+const alternating_bilinear_form = AlternatingBilinearForm
+
+#######################################
+# Show IO
+#######################################
+
+function show(io::IO, b::AlternatingBilinearForm)
+  println(io, "Alternating bilinear form, in the standard basis represented by the Gram matrix")
+  display(matrix(b))
+end
+
+#######################################
+# Generic functions
+#######################################
+
+matrix(b::AlternatingBilinearForm) = b.matrix
+is_zero(b::AlternatingBilinearForm) = is_zero(b.matrix)
+==(a::AlternatingBilinearForm, b::AlternatingBilinearForm) = a.matrix == b.matrix
+isequal(a::AlternatingBilinearForm, b::AlternatingBilinearForm) = a == b
+
+#######################################
+# Application
+#######################################
+
+function (b::AlternatingBilinearForm{T})(v::Vector, w::Vector)::T where T <: RingElem
```

Comment:
```suggestion
function (b::AlternatingBilinearForm{T})(v::Vector, w::Vector) where T <: RingElem
```
adding a type there results in the compiler inserting some `convert(T, ...)` calls, which may lead to very hard to track down bugs

(same at a few other places in this PR)